CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+Contributing to Linnia-Smart-Contracts
+=======
+
+We really appreciate and value contributions to Linnia-Smart-Contracts. Please take 5' to review the items listed below to make sure that your contributions are merged as soon as possible. 
```

Comment:
`5 minutes`  

5'  maybe be confusing for folks learning English.

Reply:
Could be just "Please take time to review the items listed below to make sure that your contributions are merged as soon as possible."